bin/p2-dsctl/main.go

```diff
@@ -55,11 +59,19 @@ var (
 
 	cmdUpdate       = kingpin.Command(CmdUpdate, "Update a daemon set.")
 	updateID        = cmdUpdate.Arg("id", "The uuid for the daemon set").Required().String()
-	updateAZ        = cmdUpdate.Flag("az", "The availability zone of the pod cluster").String()
+	updateSelector  = cmdUpdate.Flag("selector", "The node selector, uses the same syntax as the test-selector command").Default(SelectorNotSpecified).String()
```

Comment:
Would it make sense to have a DS without any node selector?  I.e., will `--selector ""` ever be valid?  If not, don't specify a default and just test for `updateSelector == ""` to see if the flag was given.

Otherwise, be sure to use [`.Placeholder()`](https://godoc.org/gopkg.in/alecthomas/kingpin.v2#FlagClause.PlaceHolder) for this flag, or your improbable default value becomes visible with `--help`.